### ISSUE TITLE
[feature request] permision handling

### ISSUE THREAD

**Issue by ValgulNecron** [moghtech/komodo#241]
Hi at the moment if i give write per on a server, and also want this user to have access to stack, builder, etc.... on this server i need to manually give him.

i would really like if either write perm on a serve give access to stack and other stuff linked to this server or a "server admin" perm that would do the same without altering the already existing level.

**Comment by mbecker20** [comment]
Yeah, I think the permissions regarding `Server` itself, vs `Deployment / Stack` that is _on_ a `Server`, is a bit awkward currently.

**Comment by mbecker20** [comment]
It doesn't really fit the permission model to make this change. It keep everything straightforward to enumerate user access to resources. There are layer-able User Groups so it doesn't get too repetitive with many users too.
